spec/models/hotel_spec.rb

```diff
@@ -4,4 +4,14 @@
   describe "relationships" do
     it { should have_many :rooms }
   end
+
+  it "should exist, and have a name and location" do
```

Comment:
In Rails, it's not really necessary for you to test the creation of the class and the access of its attributes, since Rails sets that up for us under the hood. You can just stick to relationship tests, validation tests (which are not required for ICs), and model method tests